pkg/cloudprovider/provider/vmwareclouddirector/types/types.go

```diff
@@ -46,7 +46,7 @@ type RawConfig struct {
 
 	// Network configuration.
 	Network          providerconfigtypes.ConfigVarString `json:"network"`
-	IPAllocationMode IPAllocationMode                    `json:"ipAllocationMode"`
+	IPAllocationMode IPAllocationMode                    `json:"ipAllocationMode,omitempty"`
```

Comment:
Just saying, this change has no effect on anything...

Reply:
😱

Reply:
Wouldn't it generate an error for a missing field because we strictly unmarshall this?